src/ngx_http_lua_util.c

```diff
@@ -756,6 +767,57 @@ ngx_http_lua_inject_ngx_api(lua_State *L, ngx_http_lua_main_conf_t *lmcf,
     lua_setglobal(L, "ngx");
 
     ngx_http_lua_inject_coroutine_api(log, L);
+
+#ifdef OPENRESTY_LUAJIT
+    {
+        const char buf[] =
```

Comment:
We should put the `int rc` declaration here as well. `rc` is not used elsewhere and we should limit its lifetime.